server/lib/flow-event.js

```diff
@@ -87,6 +87,7 @@ module.exports = (req, metrics, requestReceivedTime) => {
     }
 
     logFlowEvent(event, metrics, req);
+    sendToAmplitude(event, metrics, req);
```

Comment:
Is it only flow events you want to send to amplitude, or all events? If all events, I'd create a new `amplutide-collector` and write to it from [server/lib/routes/post-metrics.js](https://github.com/mozilla/fxa-content-server/blob/master/server/lib/routes/post-metrics.js).